spaceship/lib/spaceship/du/du_client.rb

```diff
@@ -111,6 +121,21 @@ def messages_picture_type_map
       }
     end
 
+    def device_resolution_map
+      # rubocop:enable Layout/ExtraSpacing
+      {
+        watch:        [[312, 390]],
+        ipad:         [[1024, 748], [1024, 768], [2048, 1496], [2048, 1536], [768, 1004], [768, 1024], [1536, 2008], [1536, 2048]],
+        ipadPro:      [[2048, 2732], [2732, 2048]],
+        iphone6:      [[750, 1334], [1334, 750]],
+        iphone6Plus:  [[1242, 2208], [2208, 1242]],
+        iphone4:      [[640, 1096], [640, 1136], [1136, 600], [1136, 640]],
+        iphone35:     [[640, 960], [640, 920], [960, 600], [960, 640]],
+        appleTV:      [[1920, 1080]],
+        desktop:      [[1280, 800], [1440, 900], [2560, 1600], [2880, 1800]]
+      }
+    end
```

Comment:
Would it be possible to have all these 2 dimensional arrays be hashes with keys "width" and "height" to make this a bit more clear?

Reply:
I was concerned with the verbosity of that option, and this looked a bit more natural to me since it's the same format `fastimage` uses to express resolutions (and is also the same format `Utilities.resolution(upload_file.file_path)` returns.

I can easily change this of course. Let me know what do you think